test/wdio_functional/page_objects/console/console_page.js

```diff
@@ -0,0 +1,146 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import expect from 'expect';
+import {
+  createTestSubjectSelectorCss,
+  createTestSubjectSelectorXpath
+} from '../../helpers/helpers';
+import BasePage from '../common/base_page';
+
+export default class ConsolePage extends BasePage {
+
+  constructor(driver) {
+    super(driver);
+    this.breadCrumbSelector = 'div.kuiLocalBreadcrumb';
```

Comment:
Out of curiosity, why are all of these properties of the `ConsolePage` class?

It feels like constants outside of the class, or just implementation details of a `getBreadCrumb()` method would make more sense.